awq/entry.py

```diff
@@ -146,6 +155,16 @@ def skip(*args, **kwargs):
                     exit(0)
             else:
                 raise NotImplementedError
+            
+            # Move the model to GPU (as much as possible) for LM evaluation
+            kwargs = {
+                "torch_dtype": torch.float16, 
+                "device_map": "auto", 
```

Comment:
Let me know if you would want me to change `device_map` to `balanced` instead of `auto`.